index.html

```diff
@@ -86,7 +77,12 @@ <h2>Leave a Message</h2>
     <h2>Messages</h2>
     <ul></ul>
   </section>
-  <footer></footer>
+  <footer>
+    <section id="connect">
```

Comment:
Would be good to add some hover styles in the CSS for these links so that it's clear that they are clickable

For example:

```css
/* styling the links for all `a` tags that are children of the tag with an id="connect" */
#connect a:hover {
    background: plum;
}
```